packages/libbpf/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+name = "libbpf"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+
+[lib]
+path = "../packages.rs"
+
+[package.metadata.build-package]
+releases-url = "https://github.com/libbpf/libbpf/releases"
+
+[[package.metadata.build-package.external-files]]
+url = "https://github.com/libbpf/libbpf/archive/refs/tags/v1.4.3.tar.gz"
+sha512 = "d03965f8abeaac840cda28fdacf9df378869fa6e5ef45d992fc5398bed5970ebedb24f20676b1779f8941b4b912bb691948e198eff87ea1d5d8e5cd95d6e06d1"
+
+# RPM BuildRequires
+[build-dependencies]
+glibc = { path = "../glibc" }
+libelf = { path = "../libelf" }
```

nit: to make it clear that `libbpf` also depends on zlib:
```suggestion
libelf = { path = "../libelf" }
libz = { path = "../libz" }
```